layouts/partials/homepage/role_slider/content.html

```diff
@@ -1,12 +1,15 @@
-<div class="tab-content">
+<div class="tab-content p-8">
```

Comment:
ditto on `@apply`ing this in the `.tab-content` selector in the main css.

Reply:
updated!